Running FedStream with: Dir=1.0, Init_Class=8, Epoch=10
Files already downloaded and verified
Files already downloaded and verified
has read
Files already downloaded and verified
Files already downloaded and verified
  0%|          | 0/100 [00:00<?, ?it/s]  1%|          | 1/100 [04:05<6:44:24, 245.10s/it]  1%|          | 1/100 [06:07<10:06:11, 367.39s/it]
Traceback (most recent call last):
  File "/home/kemove/lt/FedAlgorithm/alg/FedStream.py", line 775, in <module>
    alg.online_train()
  File "/home/kemove/lt/FedAlgorithm/alg/FedStream.py", line 738, in online_train
    global_loss_list, accuracy_list = self.alg.run(
                                      ^^^^^^^^^^^^^
  File "/home/kemove/lt/FedAlgorithm/alg/FedDyn.py", line 134, in run
    ].local_update_dyn(
      ^^^^^^^^^^^^^^^^^
  File "/home/kemove/lt/FedAlgorithm/alg/Clients.py", line 286, in local_update_dyn
    for batch in dataloader:
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/torch/utils/data/dataloader.py", line 631, in __next__
    data = self._next_data()
           ^^^^^^^^^^^^^^^^^
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/torch/utils/data/dataloader.py", line 675, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/torch/utils/data/_utils/fetch.py", line 51, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/torch/utils/data/_utils/fetch.py", line 51, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
            ~~~~~~~~~~~~^^^^^
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/torch/utils/data/dataset.py", line 348, in __getitem__
    return self.datasets[dataset_idx][sample_idx]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^
  File "/home/kemove/lt/FedAlgorithm/alg/Clients.py", line 37, in __getitem__
    data = self.trans(self.local_data[index])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/torchvision/transforms/transforms.py", line 95, in __call__
    img = t(img)
          ^^^^^^
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1535, in _call_impl
    forward_call = (self._slow_forward if torch._C._get_tracing_state() else self.forward)
                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
